lib/bundles/inspec-compliance/api.rb

```diff
@@ -21,15 +28,21 @@ def self.profiles(config)
         msg = 'success'
         profiles = JSON.parse(data)
         # iterate over profiles
-        if config['server_type'] == 'automate'
-          mapped_profiles = profiles.values.to_a.flatten
-        else
+        if config['server_type'] == 'compliance'
```

Comment:
We can definitely do that. Even after my fix, this module needs massive refactoring. It started with 1 server version, and we have essentially 2 versions Chef Compliance and 2 versions Chef Automate now. That needs to be abstracted.